.github/workflows/runner-variants.yml

```
@@ -0,0 +1,82 @@
+    tags:
+      - v*.*.*
+  release:
+    types: [ created ]
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: nestrilabs/nestri
+
+jobs:
+  bases:
+    uses: /.github/workflows/runner-bases.yml
```

actionlint (1.7.8)

reusable workflow call "/.github/workflows/runner-bases.yml" at "uses" is not following the format "owner/repo/path/to/workflow.yml@ref" nor "./path/to/workflow.yml". see https://docs.github.com/en/actions/learn-github-actions/reusing-workflows for more details

(workflow-call)